iceoryx_utils/platform/win/source/win32_errorHandling.cpp

```diff
@@ -33,7 +35,10 @@ inline int PrintLastErrorToConsole() noexcept
                       BUFFER_SIZE - 1,
                       NULL);
 
-        std::cerr << "error ( " << lastError << " ) :: " << buffer << std::endl;
+        coutMutex.lock();
+        std::cerr << "< Win32API Error > " << file << ":" << line << " { " << functionName << " } [ " << lastError
```

Comment:
Is it maybe possible to use the errorhandler here?

Reply:
no, that one is in a layer above this

Reply:
Exactly. From the iceoryx point of view we are here at the lowest possible layer it is like the operating system.